Task_2/src/model/Counter.java

```diff
@@ -0,0 +1,21 @@
+package model;
+
+public class Counter {
+    static int paragraph = 0;
+    static int sentence = 0;
+    static int word = 0;
+    static int symbol = 0;
+    static String longWord = "";
+

```

Comment:
What will be in sentence, word and symbol variables if we create more than one Text in Runner?

Reply:
program will continue count symbols, words, sentences, paragraphs.Program count all parameters in two or more texts.

Reply:
But in the [Text line 17](https://github.com/Yauhenda/task_02/pull/4/files#diff-09e5e8c684a623384cad51a265c82b48R17) you erase counter with new value. So if logic is to continue counting it is a bug. If to start new count with new text, you need to set 0 to all counter when start to parse.